Sources/tuist/main.swift

```diff
@@ -1,5 +1,13 @@
 import Foundation
-import TuistKit
+import enum Basic.ProcessEnv
+
+if CommandLine.arguments.contains("--verbose") {
+    try? ProcessEnv.setVar("TUIST_VERBOSE", value: "true")
+}
 
+import enum TuistSupport.LogOutput
+LogOutput.bootstrap()
```

Comment:
What about extending bootstrap to accept a `verbose` argument?

```swift
func bootstrap(verbose: Bool)
```

That way you don't have to set an environment variable, `TUIST_VERBOSE` than then you read from the implementation of `bootstrap`.

Reply:
The environment variable was introduced to deal with the `tuistenv` -> `tuist` relationship, since it executes it as a subprocess. I can re-look into that but this solution is one me and @kwridan came up with when we first looked into the verbose logging.